Purchases/Public/RCPurchases.m

```diff
@@ -254,7 +262,8 @@ - (instancetype)initWithAPIKey:(NSString *)APIKey
        subscriberAttributesManager:subscriberAttributesManager
                operationDispatcher:operationDispatcher
         introEligibilityCalculator:introCalculator
-                     receiptParser:receiptParser];
+                     receiptParser:receiptParser
+              purchaserInfoManager:purchaserInfoManager];
```

Comment:
this is getting a bit intense when it comes to dependencies. We might eventually want to extract this into an `environment` variable that holds these. 
The idea would be that RCPurchases has a single `environment` dependency, and the environment creates all of the others.